plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveTableProperties.java

```diff
@@ -169,7 +173,16 @@ public HiveTableProperties(
                         PARTITION_PROJECTION_LOCATION_TEMPLATE,
                         "Partition projection location template",
                         null,
-                        false));
+                        false),
+                new PropertyMetadata<>(
+                        EXTRA_PROPERTIES,
+                        "Extra table properties",
+                        typeManager.getType(TypeSignature.mapType(VARCHAR.getTypeSignature(), VARCHAR.getTypeSignature())),
+                        Map.class,
+                        null,
```

Comment:
default value can be an emptyMap

Reply:
Sorry didn't see this one - https://github.com/trinodb/trino/pull/1176#issuecomment-662753140. This comment can be reverted.